File: Wrap-up/Java/ArmstrongNumber.java
import java.util.Scanner;

public class ArmstrongNumber {
    public static void main(String... args){
        Scanner input = new Scanner(System.in);
        
        IO.println("Determine if number is an ArmstrongNumber");
        
        IO.println("Enter Number: ");
        int userNumber = input.nextInt();
        
        IO.println();
        
        int storeOne = userNumber;
        int storetwo = userNumber;
        int product = 0;
        int sum = 0;
        int counter = 0;
        
        while(storeOne != 0){
            int digits = storeOne % 10;
            
            counter++;
            
            storeOne /= 10;
        }
        
        while(storetwo != 0){
            int digits = storetwo % 10;
            
            product = (int)Math.pow(digits, counter);
            
            sum += product;
            
            storetwo /= 10;
        }
        
        if(userNumber == sum){
            IO.println(userNumber + " is an ArmstrongNumber");
        }else{
            IO.println(userNumber + " is not an ArmstrongNumber");
        }
    }
}

File: Wrap-up/python/armstrong_number.py
import math

print("Determine if number is an ArmstrongNumber")

userNumber = int(input("Enter Number: "))

print()

storeOne = userNumber
storetwo = userNumber
product = 0
sum = 0
counter = 0

while(storeOne != 0):
    digits = storeOne % 10
    
    counter += 1
    
    storeOne = storeOne // 10


while(storetwo != 0):
    digits = storetwo % 10
    
    product = math.pow(digits, counter)
    
    sum += product
    
    storetwo = storetwo // 10


if(userNumber == sum):
    print(userNumber, " is an ArmstrongNumber")
else:
    print(userNumber, " is not an ArmstrongNumber")

